cratedb_toolkit/admin/xmover/analysis/shard.py

```
@@ -836,6 +839,158 @@
+    def monitor_shards(self, table_filter: str | None, interval_in_seconds: int = 5, repeat: int = 10, n_shards: int = 40, sort_by: str = 'heat'):
+        self.table_filter = table_filter
+        self.sort_by = sort_by
+
+        self.reference_shards = {self._get_shard_compound_id(shard): shard for shard in self.analyzer.shards}
+        self.refresh_data()
+
+        console.print(Panel.fit(f"[bold blue]The {n_shards} Hottest Shards[/bold blue]"))
+
+        go_live = False
+        if go_live:
+            with Live(self.generate_shards_table(self._get_top_shards(self.latest_shards, n_shards), self.seq_deltas), refresh_per_second=4, console=console) as live_shards:
```

Ruff (0.12.2)

Line too long (173 > 120)

(E501)